app/components/cron-job.js

```diff
@@ -9,8 +9,8 @@ export default Ember.Component.extend({
   store: service(),
 
   isOverdue: function(key) {
```

Comment:
These computed properties don't appear to be using the `key` being passed in. Can they be removed?